trees/python/binary-tree.py

```diff
@@ -0,0 +1,66 @@
+from collections import deque
+
+class Node(object):
+    def __init__(self,val):
+        self.value = val
+        self.leftchild = None
+        self.rightchild = None
+
+class BinaryTree(object):
+    def __init__(self,root):
+        self.root = Node(root)
+
+    def PreOrderTraversal(self,v):
+        if v!=None:
+            print(v.value,end=" ")
+            self.PreOrderTraversal(self,v.leftchild)
+            self.PreOrderTraversal(self,v.rightchild)
+
+        return
```

Comment:
```suggestion
    def PreOrderTraversal(self, node: Node) -> None:
        if node:
            print(node.value, end=" ")
            self.PreOrderTraversal(node.leftchild)
            self.PreOrderTraversal(node.rightchild)
```
This function accepts a `Node`, not a value so `v` is not a great parameter name.  Besides, single-letter variable names make code look old-fashoned so where possible please use variable names that make code comments unnecessary.